### PR TITLE
Add bulk actions (enhancement) (#356)

### LINKED ISSUE #356: Bulk actions needed
For day-to-day blog post management, I would appreciate some bulk actions like publish/ unpublish, allow/disallow comments for a set of selected blog posts. What do others think? What else would you need?

I could contribute. -- Does anyone know hot to trigger a page reload from the DjangoCMS admin panel after changes thanks to bulk actions?

### PR DESCRIPTION
**Pull request by fsbraun**
Extended ``admin.py``  to add bulk actions to

- Publish/unpublish selected posts
- Enable/disable comments for selected posts
- Enable/disable liveblog for selected posts (only if liveblog app is installed)

Attention: This pull requests adds strings which need to be localized

Fix #356

### CODE REVIEW

File: djangocms_blog/admin.py [hunk 1]
```diff
@@ -30,6 +31,59 @@ class EnhancedModelAdminMixin(object):
         pass
 
 
+# Bulk actions for post admin
+def make_published(modeladmin, request, queryset):
+    """ Bulk action to mark selected posts as published. If
+        the date_published field is empty the current time is
+        saved as date_published.
+    """
+    for post in queryset:
```
Comment (yakky) [inline djangocms_blog/admin.py]: I'd prefer using ``queryset.update`` here and emitting a message to confirm that the posts have been published

File: djangocms_blog/admin.py [hunk 2]
```diff
@@ -30,6 +31,59 @@ class EnhancedModelAdminMixin(object):
...
+            if not post.date_published:
+                post.date_published = timezone.now()
+            post.publish = True
+            post.save()
+
+
+def make_unpublished(modeladmin, request, queryset):
+    """ Bulk action to mark selected posts as UNpublished.
+    """
+    queryset.update(publish=False)
```
Comment (yakky) [inline djangocms_blog/admin.py]: I'd prefer emitting a message to confirm the action
Ditto for the actions below

File: djangocms_blog/admin.py [hunk 3]
```diff
@@ -82,6 +136,14 @@ class PostAdmin(PlaceholderAdminMixin, FrontendEditableAdminMixin,
     search_fields = ('translations__title',)
     date_hierarchy = 'date_published'
     raw_id_fields = ['author']
+    actions = [
```
Comment (yakky) [inline djangocms_blog/admin.py]: I'd like to have the actions as method of the admin class as they are not going to be reused elsewhere

**Comment by yakky** [comment]
Nice and needed actions

File: djangocms_blog/admin.py (hunk 3 above)
Comment (fsbraun) [inline djangocms_blog/admin.py]: Yep. Good idea.

File: djangocms_blog/admin.py (hunk 2 above)
Comment (fsbraun) [inline djangocms_blog/admin.py]: I'll use ``contrib.messages``.

File: djangocms_blog/admin.py (hunk 1 above)
Comment (yakky) [inline djangocms_blog/admin.py]: how about something like:
```
queryset.filter(date_published__isnull=True).update(date_published=now(), publish=True)
queryset.filter(date_published__isnull=False).update(publish=True)
```

Comment (fsbraun) [inline djangocms_blog/admin.py]: Well, maybe I did not think that through thoroughly... Will use a ``queryset.update``.

Comment (yakky) [inline djangocms_blog/admin.py]: No worries. That's what code review are for!

**Comment by fsbraun** [comment]
Process question: Can I add enhancements of #357 to the same pull request or should I create a new one?

**Comment by yakky** [comment]
@fsbraun definitely a new PR for #357

File: djangocms_blog/admin.py (hunk 3 above)
Comment (yakky) [inline djangocms_blog/admin.py]: Please check this https://docs.djangoproject.com/en/1.10/ref/contrib/admin/actions/#actions-as-modeladmin-methods for how to implement method-based actions

**Comment by fsbraun** [comment]
Implemented requested changes.
Not sure why I get a red flag form coveralls and codecov though ...

**Comment by yakky** [comment]
@fsbraun as the new methods have no test, the coverage has decreased. I'll ponder a bit on the best way to test them

**Comment by fsbraun** [comment]
@yakky anything I can do at this point?

**Comment by yakky** [comment]
Sorry @fsbraun I completely missed this. LGTM, merging
Thanks for your contribution!

**Pull request merged**